src/Temporalio/Testing/WorkflowEnvironmentStartLocalOptions.cs

```diff
@@ -0,0 +1,29 @@
+namespace Temporalio.Testing
+{
+    /// <summary>
+    /// Options for <see cref="WorkflowEnvironment.StartLocalAsync" />.
+    /// </summary>
+    public class WorkflowEnvironmentStartLocalOptions : Client.TemporalClientConnectOptions
```

Comment:
Override `Clone` and make sure to clone the `TemporaliteOptions` options too (same for `WorkflowEnvironmentStartTimeSkippingOptions`)

Reply:
Fixed